framework/src/modules/nft/method.ts

```diff
@@ -34,4 +38,42 @@ export class NFTMethod extends BaseMethod {
 		this._interoperabilityMethod = interoperabilityMethod;
 		this._internalMethod = internalMethod;
 	}
+
+	public async getNFTOwner(methodContext: ImmutableMethodContext, nftID: Buffer): Promise<Buffer> {
+		const nftStore = this.stores.get(NFTStore);
+
+		const nftExists = await nftStore.has(methodContext, nftID);
+
+		if (!nftExists) {
+			throw new Error('NFT substore entry does not exist');
+		}
+
+		const data = await nftStore.get(methodContext, nftID);
+
+		return data.owner;
+	}
+
+	public async getLockingModule(
+		methodContext: ImmutableMethodContext,
+		nftID: Buffer,
+	): Promise<string> {
+		const nftStore = this.stores.get(NFTStore);
+
+		const nftExists = await nftStore.has(methodContext, nftID);
+
+		if (!nftExists) {
+			throw new Error('NFT substore entry does not exist');
+		}
```

Comment:
I think this check is redundant since it's also being checked below in `getNFTOwner`

Reply:
[Modified under the commit](https://github.com/LiskHQ/lisk-sdk/pull/8503/commits/505e3d8c99fdf8b982586862dae6d551deff412d)